src/test/java/lv/ctco/javaschool/auth/boundary/AuthenticationApiTest.java

```diff
@@ -32,9 +32,16 @@ void testReturnUserDto() {
         user.setPhone("1234567");
         user.setEmail("a@b.com");
         Mockito.when(userStore.getCurrentUser()).thenReturn(user);
-        UserLoginDto dto = authenticationApi.returnUserDto();
-        assertEquals(user.getUsername(), dto.getUsername());
-        assertEquals(user.getEmail(), dto.getEmail());
-        assertEquals(user.getPhone(), dto.getPhone());
+
+        UserLoginDto dto = new UserLoginDto();
+        dto.setUsername(user.getUsername());
+        dto.setPhone(user.getPhone());
+        dto.setEmail(user.getEmail());
+        Mockito.when(userStore.convertToDto(user)).thenReturn(dto);
+
+        UserLoginDto resultDto = authenticationApi.returnUserDto();
+        assertEquals(user.getUsername(), resultDto.getUsername());
```

Comment:
Consider using `assertThat()` instead of `assertEquals()` - it will give clearer messages in case of failures, and more flexibility. See this page for more information: https://confluence.ctco.lv/display/DEVCOM/Hamcrest+Matchers

Reply:
Done